policy/DecamMapper.paf

```diff
@@ -421,57 +385,25 @@ datasets: {
         tables:        raw
         tables:        raw_visit
     }
+    # dataset defined in obs_base modified in this mapper as follows:
     deepCoadd_forced_metadata: {
         template:      "deepCoadd-results/forcedPhotCoadd_metadata/%(filter)s/%(tract)d/%(patch)s.boost"
-        python:        "lsst.daf.base.PropertySet"
-        persistable:   "PropertySet"
-        storage:       "BoostStorage"
     }
+    # dataset defined in obs_base modified in this mapper as follows:
     deepCoadd_forced_src: {
         template:      "deepCoadd-results/%(filter)s/%(tract)d/%(patch)s/forcedSrc-%(filter)s-%(tract)d-%(patch)s.fits"
-        python:        "lsst.afw.table.SourceCatalog"
-        persistable:   "ignored"
-        storage:       "FitsCatalogStorage"
     }
+    # dataset defined in obs_base modified in this mapper as follows:
```

Comment:
I will change this in obs_base.  this should be lsst.pipe.tasks.imageDifference.ImageDifferenceConfig